api/product/descutoutservice.py

```diff
@@ -496,6 +496,86 @@ def test_api_help(self):
         print(req.text)
         print(req.json()['job'])
 
+    def get_job_by_ra_dec(self, ra, dec):
```

Comment:
@felipelm  Talvez fosse uma boa encapsular esses metodos dentro de uma Classe 
pq são métodos que só serão usados quando for fazer cutout de apenas um objeto. 
não tenho uma boa ideia para nome da classe rsss.